src/chrome/content/rules/RateYourMusic.com.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+	Supported Languages (from a combination of their Wikipedia page and this page: https://rateyourmusic.com/wiki/RYM:Languages):
+		Default:
+			English (no subdomain provided or needed)
+
+		Has a subdomain:
+			French: fr. (supports https)
+			German: de. (supports https)
+			Polish: pl. (supports https)
+			Russian: ru. (supports https)
+
+		Couldn't find a subdomain:
+			Dutch
+			Finnish
+			Italian
+			Japanese
+			Portuguese
+			Spanish
+			Turkish
+-->
+<ruleset name="RateYourMusic.com">
+	<!-- request with www. is rewritten automatically to remove www. so we don't need a rule for that -->
```

Comment:
Thanks!

Please create new files and separate PRs for those hosts. You can still making a comment to show the relationship between them.